flipperdevices/flipperzero-firmware
# FuriHal: explicitly pull display pins at early init stage, move PUPD config to early stage

# What's new

- FuriHal: explicitly pull display pins at early init stage, move PUPD config to early stage

# Verification 

- Ensure that all boot modes are working
- Ensure that all update modes are working 

# Checklist (For Reviewer)
